### DISCUSSION TITLE
Παραδοτέο 7 - 2018171

### DISCUSSION THREAD

**Discussion by perikarananou** [courses-ionio/sw#1528] [Show and tell]
# 7. 	Συμμετοχικό περιεχόμενο B1:

Για το 7ο παραδοτέο πρόσθεσα στο site μία μελέτη περίπτωσης που αφορά το tablet PC 2001.

* [images](https://github.com/perikarananou/images): [tabletPC](https://github.com/perikarananou/images/blob/master/tabletPC.jpg), [tabletPCmin](https://github.com/perikarananou/images/blob/master/tabletPCmin.jpg)
* [_includes](https://github.com/perikarananou/site/tree/master/_includes): [tabletPC](https://github.com/perikarananou/site/blob/master/_includes/tabletPC.md)
* [_case-study](https://github.com/perikarananou/site/tree/master/_case-study): [tabletPC](https://github.com/perikarananou/site/blob/master/_case-study/tabletPC.md)
* [Η αναφορά μου](https://github.com/perikarananou/sw/blob/2018171/projects/2018171/README.md)

**Comment by aggelos2000430** [comment]
Χαίρετε συνάδελφε, στο header του discussion υπάρχει ένα τυπογραφικό λάθος μιας και είναι:  7. Γραμμή εντολών (custom static blog generator) αντί για: 7. Συμμετοχικό περιεχόμενο Β1. Επίσης σοφό θα ήταν να παραθέσεις συνδέσμους που να οδηγούν στο περιεχόμενο που πρόσθεσες στο site!

**Reply by perikarananou** [reply]
Σε ευχαριστώ πολύ για την παρατήρηση σου το διόρθωσα 🙂
